### PR TITLE
fix: pythonrt now builds on windows

### PR DESCRIPTION
**Pull request by itayd**

### CODE REVIEW

File: runtimes/pythonrt/local_runner_windows.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//go:build windows
+
+package pythonrt
+
+import "os/exec"
+
+func setCmdSysAttrPGID(cmd *exec.Cmd) {}
```
Comment (tebeka) [inline runtimes/pythonrt/local_runner_windows.go:7]: Does this mean that on windows if `ak` fails the Python process will continue to run? @efiShtain ?

File: runtimes/pythonrt/pythonrt_test.go [hunk 2]
```diff
@@ -16,13 +16,14 @@ import (
...
-	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
-	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 	"go.uber.org/zap"
 	"golang.org/x/net/http2"
 	"golang.org/x/net/http2/h2c"
 
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+
```
Comment (tebeka) [inline runtimes/pythonrt/pythonrt_test.go:26]: Where are all this formatting changes come from?

**Review by tebeka — COMMENTED** [review]
I'd love to see @efiShtain answering my comment.
Also: Is there a product requirement to run on windows?

File: runtimes/pythonrt/local_runner_windows.go (hunk 1 above)
Comment (daabr) [reply runtimes/pythonrt/local_runner_windows.go:7]: +1. Maybe it's better to have a Windows version of `Start` that ensures the runner is shut down without PGID?

File: runtimes/pythonrt/pythonrt_test.go (hunk 2 above)
Comment (daabr) [reply runtimes/pythonrt/pythonrt_test.go:26]: AFAIK the Go formatter can handle correctly both 2-block and 3-block import sections, it just doesn't know that *our* default is 3 blocks. All the rest below is automatic for gofumpt.

**Pull request closed without merging**